Frameworks/Foundation/NSURLProtocol_WinHTTP.mm

```diff
@@ -243,20 +244,35 @@ - (instancetype)initWithRequest:(NSURLRequest*)request
     };
 }
 
+// Helper function that executes a block on the client thread - used for client callbacks
+static void __dispatchClientCallback(NSURLProtocol_WinHTTP* protocol, void (^callbackBlock)()) {
+    if (protocol->_clientThread) {
+        [protocol performSelector:@selector(_invokeBlock:) onThread:protocol->_clientThread withObject:callbackBlock waitUntilDone:NO];
```

Comment:
1) don't our NSBlocks not inherit from NSObject and thus not have that function?
2) our NSBlock has a comment on it that we ought to be reducing internal usage of the function. I think that's silly, but...